taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/config/OkHttpProperties.java

```diff
@@ -0,0 +1,29 @@
+package pro.taskana.adapter.systemconnector.camunda.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "okhttp")
+public class OkHttpProperties {
+
+  private long connectionTimeout = 2_000;
```

Comment:
Why did you choose this values for the timeouts?

Reply:
Good question, there was no spec :)

The values come from a gut feeling.

Personally, I think 2 seconds to connect is more than enough. If it takes longer, you should have a look at your network or check where the request is getting stuck. (Normally the connection should be established within 500ms).

Well, and ReadTimeout is such a feeling. @gitgoodjhe can probably give you a better answer what a good default value is for reading data from Camunda.

Does it answer your question?